pkg/virt-launcher/virtwrap/network/podinterface_test.go

```diff
@@ -287,15 +292,15 @@ var _ = Describe("Pod Network", func() {
 		Expect(err).To(BeNil())
 	}
 
-	TestRunPlug := func(driver BindMechanism) {
-		err := driver.discoverPodNetworkInterface()
+	TestRunPlug := func(bindMechanism BindMechanism) {
+		err := bindMechanism.discoverPodNetworkInterface()
```

Comment:
This rename has nothing to do with this change, does it ? Your commit message is:

"Making the handler a member of the podNICImpl struct enables us to properly
mock it in hte unit tests"

And looking below looks like most of the changes in this commit originate from this rename. 

I think you should either leave renaming out of the commit, or update the commit msg.

Reply:
Going back to "driver" to make the PR smaller, the renaming can be done at a follow up if needed